styles/style.css

```diff
@@ -260,3 +260,96 @@ button {
   border-width: 2px;
   color: var(--primary-clr);
 }
+
+/* Pair Programing */
+input,
+textarea {
+  border-style: solid;
+  background-color: #fff;
+  border-radius: 4px;
+  color: var(--primary-clr);
+  font-weight: var(--fw-400);
+  font-size: var(--fs-14);
+  padding-left: 1em;
+  width: 100%;
+}
+
+input::placeholder,
+textarea::placeholder {
+  color: #7d7c84;
+}
+
+input {
+  border: 1px solid #dfdfe0;
+  padding-bottom: 0.5em;
+  padding-top: 2em;
+}
+
+textarea {
+  padding-top: 1em;
+  margin-bottom: 1.25em;
+}
+
+input:focus,
+textarea:focus {
+  outline: none;
+  border-color: var(--primary-clr);
+}
+
+#contact {
+  background-color: #f9f9fa;
+  background-image: url("../assets/images/image_geometry_1.svg");
+  background-position: left top;
+  background-repeat: no-repeat;
+  padding-top: 6.625em;
+  display: flex;
+  flex-direction: column;
+}
+
+.social-links {
+  background: var(--bg-blue);
+  margin-top: 4em;
+  padding-top: 2em;
+}
```

Comment:
- [ ] According to the requirements You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9). So kindly change the background color to `#1A2236` to look similar to the original design 💯👌

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/186660910-56aa805c-cedb-44aa-9cf9-b5ed1fb2a596.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/186660926-29ea9ab0-8aa7-4ea3-bb00-926fb6c201fd.PNG" width="500" /> |

Reply:
Thank you @Bondok6, I'll do so.